pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.baidu.hugegraph</groupId>
     <artifactId>hugegraph-common</artifactId>
-    <version>1.8.8</version>
+    <version>1.8.9</version>
```

Comment:
line 290 `<Implementation-Version>1.8.8.0</Implementation-Version> ` also need modify to 1.8.9.0